crates/biome_lsp/src/session.rs

```diff
@@ -588,13 +590,14 @@ impl Session {
         scan_kind: ScanKind,
     ) {
         let session = self.clone();
-        let scan_project = move || {
+
+        spawn_blocking(move || {
             let result = session
                 .workspace
                 .scan_project_folder(ScanProjectFolderParams {
                     project_key,
                     path: Some(project_path),
-                    watch: true,
+                    watch: false,
```

Comment:
Why wouldn't we want to watch it? This is the only place we set it to `true` (outside tests).